EmployeeContainer.py

```diff
@@ -1,311 +1,345 @@
-        return True
-
-
-class EmployeeAdmin(EmployeeSelf):
-
-    def setEmpID(self, val: str) -> Boolean:
-        super()._employee.Emp_ID = val
-        return True
-
-    def setDOB(self, val: datetime) -> Boolean:
-        super()._employee.D_O_B = val
-        return True
-
-    def setSSNum(self, val: int) -> Boolean:
-        super()._employee.SS_num = val
-        return True
-
-    def setStartDate(self, val: datetime) -> Boolean:
-        super()._employee.Start_Date = val
-        return True
-
-    def setEndDate(self, val: datetime) -> Boolean:
-        super()._employee.End_Date = val
-        return True
-
-    def setPermissionLevel(self, val: int) -> Boolean:
-        super()._employee.Permission_level = val
-        return True
-
-    def setTitle(self, val: str) -> Boolean:
-        super()._employee.Title = val
-        return True
-
-    def setDept(self, val: str) -> Boolean:
-        super()._employee.Dept = val
-        return True
-
-    def setOfficeEmail(self, val: str) -> Boolean:
-        super()._employee.Office_email = val
-        return True
-
-    def setActive(self, val: Boolean) -> Boolean:
-        super()._employee.active = val
-        return True
-
-    def setHomePhone(self, val: str) -> Boolean:
-        super()._employee.home_phone = val
-        return True
-
-    def setHomeEmail(self, val: str) -> Boolean:
-        super()._employee.home_email = val
-        return True
-
-    def setPayType(self, val: str) -> Boolean:
-        super()._employee.pay_type = val
-        return True
-
-    def setBankInfo(self, val: str) -> Boolean:
-        super()._employee.bank_info = val
-        return True
+    permissionList = MappingProxyType({'name': [True, False], 'address': [False, False], 'office_phone': [True, False], 'Emp_ID': [True, False], 'D_O_B': [False, False], 'SS_num': [False, False], 'Start_Date': [True, False], 'End_Date': [True, False], 'Permission_level': [False, False], 'Title': [True, False], 'Dept': [True, False], 'Office_email': [
+        True, False], 'hashed_password': [False, False], 'active': [True, False], 'permitted_lock_on': [True, False], 'home_email': [False, False], 'home_phone': [False, False], 'pay_type': [False, False], 'bank_info': [False, False], 'route': [False, False], 'salary': [False, False], 'hourly': [False, False], 'commission': [False, False]})
 
-    def setRoute(self, val: str) -> Boolean:
-        super()._employee.route = val
-        return True
 
-    def setSalary(self, val: str) -> Boolean:
-        super()._employee.salary = val
-        return True
+class EmployeeSelf(EmployeeContainer):
+    permissionList = MappingProxyType({'name': [True, False], 'address': [True, False], 'office_phone': [True, False], 'Emp_ID': [True, False], 'D_O_B': [True, False], 'SS_num': [True, False], 'Start_Date': [True, False], 'End_Date': [True, False], 'Permission_level': [True, False], 'Title': [True, False], 'Dept': [True, False], 'Office_email': [
```

Comment:
Shouldn't the employee be able to edit certain values on themselves?
![image](https://user-images.githubusercontent.com/4956867/199525062-3be0bc94-9911-4816-8edf-c134a26e6289.png)